types/react-relay/index.d.ts

```diff
@@ -17,6 +17,8 @@
 export {
     commitLocalUpdate,
     commitRelayModernMutation as commitMutation,
+    CRelayContext,
+    Environment,
```

Comment:
These are exports, not imports, hence the CI failure.

Reply:
yes lol. I tried to cowboy my way from the github editor :P